examples/src/components/HeatMapExample.jsx

```diff
@@ -0,0 +1,74 @@
+// (C) 2007-2018 GoodData Corporation
+import React, { Component } from 'react';
+import { HeatMap } from '@gooddata/react-components';
```

Comment:
We asked our documentators and all agree we should rename HeatMap to Heatmap to keep it consistent with Treemap. Especially name of react components exposed to cutomers